shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -1311,4 +1314,12 @@ object GlutenConfig {
         "'spark.bloom_filter.max_num_bits'")
       .longConf
       .createWithDefault(4194304L)
+
+  val COLUMNAR_VELOX_FILE_HANDLE_CACHE_ENABLED =
+    buildConf("spark.gluten.sql.columnar.backend.velox.fileHandleCacheEnabled")
+      .internal()
+      .doc("Disables caching if false. File handle cache should be disabled " +
+        "if files are not immutable, i.e. file content may change while file path stays the same.")
```

Comment:
Suggestion: "are not immutable" -> "are mutable", to make it straightforward to reader.